test/test_helper.rb

```diff
@@ -8,8 +8,10 @@
 require_relative '../config/environment'
 require 'rails/test_help'
 require 'minitest/mock'
+require 'webmock/minitest'
 
 class ActiveSupport::TestCase
+  WebMock.allow_net_connect!
```

Comment:
Doesn't this make `webmock` gem useless? Perhaps it's better to allow some concrete URL on a per-test basis? I wonder what you want to archive by this, still.

Reply:
Actually it doesn't, I did this intentionally. I didn't want webmock to interfere with other tests since it's only being used in `SharedFooterFetcherJobTest` and with current setup stubs are working great. But if you see this as a concern, I could take a look again, sure.